addons/web/static/tests/views/list_tests.js

```diff
@@ -1501,6 +1501,119 @@ QUnit.module('Views', {
         list.destroy();
     });
 
+    QUnit.test('editable list: headers widths are only frozen on any interraction if the list is visible', async function (assert) {
```

Comment:
interaction

Reply:
Actually, this is an implementation detail. I prefer to name test according to the scenario they reproduce. So in this case, an editable list view in an initially unselected notebook page